### DISCUSSION TITLE
Yo Leather, do you mind if me and my friend use this engine for an fnf game we are makin in godot called funkville?

### DISCUSSION THREAD

**Discussion by ShygeeGit** [Leather128/LeatherEngineGodot#12] [General]
Funkville is a club penguin like game and we wanna include a place to rap battle people and since your godot fnf engine is arguably the best out there, i was wondering if we could use it, with credit ofc

**Comment by Leather128** [comment]
i mean sure ig, project is open source and all that so anyone should feel free to go wild anyways (although asking and crediting is very appreciated of course) :+1:
